### DISCUSSION TITLE
Componentes VUEX

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#550] [Q&A]
- Componente para combinar sabores dependiendo de sus preferencias

- Componente para combinar adornos dependiendo de sus preferencias

- Componente de formulario debe de contener los datos de contacto del cliente que son Nombre, Teléfono, Correo Electrónico, Descripción general del pastel y la selección de sabores y adornos

_______________________________________

Buenas, aquí serían los únicos componentes dónde se utilizará JS del Cliente.

Aquí lo que entendí es que por ejemplo combinar  sabores y adornos tendrán su componente cada uno y serian checkbox. 

Pero el del formulario lo vuelve a repetir, quiere decir que en el componente formulario llamará los componentes de combinar adornos y y sabores 

________________________________________

De igual forma en el pastelero será el único componente como Js

- Componente donde aparezcan los pedidos que se han realizado con los datos del formulario

Aquí de lo que se guardo del formulario se mostrará no?, Pero si no hay back end como se guardara (?) En un arreglo o objetos (?) De igual forma cuando se reinicie me imagino que se borra

**Comment by codingcodax** [comment]
Hola.
Conforme a tu primera duda de en el formulario llama a los componentes de combinar adornos y sabores, yo también lo entendí así, aunque no estoy seguro. 

En cuanto a tu duda de guardar la información lo puedes guardar en el `local storage`, ya que no hay nada de autenticación, no hay necesidad de hacerlo con las cookies. Y sí, tiene que ser un array para que lo hagas con un `.map()`.

**Reply by GloriaCoralCerecedo** [reply]
Oh bueno gracias, si es que apenas veré como hacelo jajaja

**Reply by richirrim** [reply]
Talvez ahí hay que usar el Store de Vuex para almacenar la info y de ahí obtenerlos desde el dashboard del pastelero.

**Comment by richirrim** [comment]
He mira respondiendo a tu duda, cuando hagas clic en el botón de tu formulario se va lanzar el evento clic y una vez el evento se lanza mandara a llamar alguna función que tengas asociada a ese evento, dicha función obtendrá  la info de tu Form y aparte la info de los checkbox seleccionados y todo eso junto lo enviaras y lo pintaras en la  interfaz del pastelero.

**Reply by GloriaCoralCerecedo** [reply]
Si gracias jaja eso ando checando como hacerlo
